pkg/interface/chat/src/js/components/lib/chat-input.js

```diff
@@ -505,29 +476,4 @@ export class ChatInput extends Component {
       </div>
     );
   }
-
-  render() {
-    const { props, state } = this;
-
-    let writePermission = props.permissions[`/chat${props.station}/write`];
-    if (writePermission) {
-      if (writePermission.kind === 'black') {
-        // black
-        if (writePermission.who.has(window.ship)) {
-          return this.readOnlyRender();
-        } else {
-          return this.writeAccessRender();
-        }
-      } else if (writePermission.kind === 'white') {
-        // white
-        if (writePermission.who.has(window.ship)) {
-          return this.writeAccessRender();
-        } else {
-          return this.readOnlyRender();
-        }
-      }
-    } else {
-      return this.writeAccessRender();
-    }
-  }
```

Comment:
I guess we don’t have read only rooms anymore? Thinking. Thinking. Wait, what if you ban someone? What do they see?

Reply:
We currently show a “you are disconnected from this chatroom” box in that case

Reply:
Ok then whatev.